logging/cluster-logging-release-notes.adoc

```diff
@@ -11,8 +11,12 @@ toc::[]
 .Logging Compatibility
 The {logging-title} is provided as an installable component, with a distinct release cycle from the core {product-title}. The link:https://access.redhat.com/support/policy/updates/openshift#logging[Red Hat OpenShift Container Platform Life Cycle Policy] outlines release compatibility.
 
+//include::modules/cluster-logging-rn-5.5.1.adoc[leveloffset=+1]
```

Comment:
Commented out intentionally, right?

Reply:
Yes, originally 5.5.1 was going to be included in this PR, but its release was moved out. Commented it out & left the file for ease of use re 5.5.1 releasing.